scripts/exgfs_wave_post_gridded_sbs.sh

```diff
@@ -217,219 +218,193 @@ source "${USHgfs}/preamble.sh"
-    export GRDIDATA=${DATA}/output_$YMDHMS
+  export GRIBDATA=${DATA}/output_$YMDHMS
+  export GRDIDATA=${DATA}/output_$YMDHMS
 
 # Gridded data (main part, need to be run side-by-side with forecast
-    
-    if [ $fhr = $fhrg ]
-    then
-      for wavGRD in ${waveGRD}; do
-        gfile="${COMIN_WAVE_HISTORY}/${WAV_MOD_TAG}.out_grd.${wavGRD}.${YMD}.${HMS}"
-        if ! wait_for_file "${gfile}" "${sleep_interval}" "${iwaitmax}"; then
-          echo " FATAL ERROR : NO RAW FIELD OUTPUT FILE out_grd.${grdID}"
-          echo "${WAV_MOD_TAG} post ${grdID} ${PDY} ${cycle} : field output missing."
-          err=3; export err; "${errchk}"
-          exit "${err}"
-        fi
-        ${NLN} "${gfile}" "./out_grd.${wavGRD}"
-      done
-      
-      if [ "$DOGRI_WAV" = 'YES' ]
-      then
-        nigrd=1
-        for grdID in $waveinterpGRD
-        do
-          ymdh_int=$($NDATE -${WAVHINDH} $ymdh); dt_int=3600.; n_int=9999 ;
-          echo "${USHgfs}/wave_grid_interp_sbs.sh $grdID $ymdh_int $dt_int $n_int > grint_$grdID.out 2>&1" >> ${fcmdigrd}.${nigrd}
-          if [ "$DOGRB_WAV" = 'YES' ]
-          then
-            gribFL=\'$(echo ${OUTPARS_WAV})\'
-            case $grdID in
-              glo_15mxt) GRDNAME='global' ; GRDRES=0p25 ; GRIDNR=255  ; MODNR=11 ;;
-              reg025) GRDNAME='global' ; GRDRES=0p25 ; GRIDNR=255  ; MODNR=11 ;;
-              glo_025) GRDNAME='global' ; GRDRES=0p25 ; GRIDNR=255  ; MODNR=11 ;;
-              glo_100) GRDNAME='global' ; GRDRES=1p00 ; GRIDNR=255  ; MODNR=11 ;;
-              glo_200) GRDNAME='global' ; GRDRES=2p00 ; GRIDNR=255  ; MODNR=11 ;;
-              glo_500) GRDNAME='global' ; GRDRES=5p00 ; GRIDNR=255  ; MODNR=11 ;;
-              glo_30mxt) GRDNAME='global' ; GRDRES=0p50 ; GRIDNR=255  ; MODNR=11 ;;
-              glo_30m) GRDNAME='global' ; GRDRES=0p50 ; GRIDNR=255  ; MODNR=11 ;;
-              at_10m) GRDNAME='atlocn' ; GRDRES=0p16 ; GRIDNR=255  ; MODNR=11   ;;
-              ep_10m) GRDNAME='epacif' ; GRDRES=0p16 ; GRIDNR=255  ; MODNR=11   ;;
-              wc_10m) GRDNAME='wcoast' ; GRDRES=0p16 ; GRIDNR=255  ; MODNR=11   ;;
-              ak_10m) GRDNAME='alaska' ; GRDRES=0p16 ; GRIDNR=255  ; MODNR=11   ;;
-            esac
-            echo "${USHgfs}/wave_grib2_sbs.sh $grdID $GRIDNR $MODNR $ymdh $fhr $GRDNAME $GRDRES $gribFL > grib_$grdID.out 2>&1" >> ${fcmdigrd}.${nigrd}
-          fi
-          echo "${GRIBDATA}/${fcmdigrd}.${nigrd}" >> ${fcmdnow}
-          chmod 744 ${fcmdigrd}.${nigrd}
-          nigrd=$((nigrd+1))
-        done
+
+  if [ $fhr = $fhrg ]
+  then
+    for wavGRD in ${waveGRD}; do
+      gfile="${COMIN_WAVE_HISTORY}/${WAV_MOD_TAG}.out_grd.${wavGRD}.${YMD}.${HMS}"
+      if ! wait_for_file "${gfile}" "${sleep_interval}" "${iwaitmax}"; then
+        echo " FATAL ERROR : NO RAW FIELD OUTPUT FILE out_grd.${grdID}"
+        echo "${WAV_MOD_TAG} post ${grdID} ${PDY} ${cycle} : field output missing."
+        err=3; export err; "${errchk}"
+        exit "${err}"
```

Comment:
Should no longer need to wait for files anymore. Either is is there or fail.
```suggestion
      if [[ -s "${gfile}" ]]; then
        echo " FATAL ERROR : NO RAW FIELD OUTPUT FILE ${gfile}"
        err=3; export err; "${errchk}"
        exit "${err}"
```